lib/commands/animation.js

```diff
@@ -15,9 +15,13 @@ import { ANIMATION_SETTING_ACTION, ANIMATION_SETTING_RECEIVER } from '../constan
  * @param {boolean} on - True to enable and false to disable it.
  */
 export async function setAnimationState (on) {
-  await this.checkBroadcast([
-    '-a', ANIMATION_SETTING_ACTION,
-    '-n', ANIMATION_SETTING_RECEIVER,
-    '--es', 'setstatus', on ? 'enable' : 'disable'
-  ], `${on ? 'enable' : 'disable'} animation`);
+  await this.adb.getApiLevel() >= 26
+    ? await this.adb.setAnimationScale(on ? 1 : 0)
+    // Please delete AnimationSettingHandler.java implementation also to avoid non-sdk https://developer.android.com/guide/app-compatibility/restrictions-non-sdk-interfaces
+    // error in settings app for Android 9 and over when we drop api level 25 and lower.
+    : await this.checkBroadcast([
+      '-a', ANIMATION_SETTING_ACTION,
+      '-n', ANIMATION_SETTING_RECEIVER,
+      '--es', 'setstatus', on ? 'enable' : 'disable'
```

Comment:
maybe it makes sense to extract the result of `on ? 'enable' : 'disable'` to a variable to avoid duplication